spec/paperclip/file_command_content_type_detector_spec.rb

```diff
@@ -11,6 +11,12 @@
     tempfile.close
   end
 
+  it 'returns a meaningful content type for open xml spreadsheets' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.